packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -43,46 +119,217 @@ const processDirectoryRecursively = (currentPath: string) => {
+      fs.writeFileSync(filePath, modifiedContent, "utf8");
+      hasReplaced = true;
+      console.log(`sites-components imports replaced in: ${filePath}`);
+    }
+  };
+  processDirectoryRecursively(source, operation);
+  return hasReplaced;
+};
+
+/**
+ * Replaces imports for react-components with pages-components
+ * @param source the src folder
+ * @return hasReplaced
+ */
+export const replaceReactComponentsImports = (source: string): boolean => {
+  let hasReplaced = false;
+  const operation = async (filePath: string) => {
+    const fileContent = fs.readFileSync(filePath, "utf8");
+    if (fileContent.match(reactComponentsRegex)) {
+      const modifiedContent = fileContent.replace(
+        reactComponentsRegex,
+        pagesComponentsReplacement
+      );
+      fs.writeFileSync(filePath, modifiedContent, "utf8");
+      hasReplaced = true;
+      console.log(`react-components imports replaced in: ${filePath}`);
+    }
+  };
+  processDirectoryRecursively(source, operation);
+  return hasReplaced;
+};
+
+/**
+ * Removes old fetch import that is no longer used
+ * @param root folder that contains package.json and tsconfig.json
+ */
+export const removeFetchImport = (root: string) => {
+  let hasRemoved = false;
+  const project = new Project({
+    tsConfigFilePath: path.resolve(root, "tsconfig.json"),
+  });
+  const sourceFiles = project.getSourceFiles();
+  for (let i = 0; i < sourceFiles.length; i++) {
+    const sourceFile = sourceFiles[i];
+    const importDeclarations = sourceFile.getImportDeclarations();
+    for (let j = 0; j < importDeclarations.length; j++) {
+      const importDeclaration = importDeclarations[j];
+      const namedImports = importDeclaration.getNamedImports();
+      for (let k = 0; k < namedImports.length; k++) {
+        const namedImport = namedImports[k];
+        if (namedImport.getName() === "fetch") {
+          if (namedImports.length == 1) {
+            importDeclaration.remove();
+          } else {
+            namedImport.remove();
+          }
+          hasRemoved = true;
+          console.log(
+            `Removed legacy fetch import in: ${sourceFile.getFilePath()}`
+          );
+          break;
+        }
+      }
+    }
+  }
+  if (hasRemoved) {
+    project.saveSync();
   }
 };
 
-// Function to update package.json scripts
-const updatePackageScripts = (targetDirectory: string) => {
-  const packageJsonPath = path.resolve(targetDirectory, "package.json");
+/**
+ * Updates package scripts to include dev, prod, and build:local commands
+ * @param root folder that includes package.json
+ */
+export const updatePackageScripts = (root: string) => {
+  const packageJsonPath = path.resolve(root, "package.json");
 
   try {
     const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf-8"));
     const scripts = packageJson.scripts;
     scripts.dev = "pages dev";
     scripts.prod = "pages prod";
     scripts["build"] = "pages build";
+    scripts["build:local"] = undefined;
```

Comment:
Doing this means we need to update their buildCmd to use build instead of build:local. Do we do that already?

Reply:
Yup
https://github.com/yext/pages/blob/main/packages/pages/src/upgrade/migrateConfig.ts#L48